app/utils/AddressHelper.scala

```diff
@@ -76,7 +76,7 @@ object AddressHelper {
   }
 
   def formatAddressResponse(address: AddressResponse): HtmlContent = {
```

Comment:
I can see there is an existing test for this method but considering that it didn't catch the bug then perhaps it requires amendment as it might not have been setup correctly.

Reply:
Oh yeah
`    Address(line1, line2, line3, line4, safePostcode, Country.GB)
`
only GB - perhaps this was only used for UK addresses?